src/main/java/org/primefaces/renderkit/SelectManyRenderer.java

```diff
@@ -39,7 +39,11 @@ public void decode(FacesContext context, UIComponent component) {
         Map<String, String[]> params = context.getExternalContext().getRequestParameterValuesMap();
 
         if (params.containsKey(submitParam)) {
-            selectMany.setSubmittedValue(params.get(submitParam));
+            String[] submittedValues = params.get(submitParam);
+            if (isDisabledSelectItemSubmitted(context, selectMany, (Object[]) getValues(selectMany), submittedValues)) {
+                return;
```

Comment:
Hmm should it return or should it throw an exception because the client side has been manipulated?  if we don't throw an exception it will just look like silently nothing is happening?